### DISCUSSION TITLE
包含标点符号的自造词

### DISCUSSION THREAD

**Discussion by shidejie** [rime/home#1725] [Q&A]
当我输完“**_明知山有虎，偏向虎山行_**”的时候，我希望rime能够造出该词句，这样我下次只要输入jtmt（五笔为例）即可输入该词句。
rime是否有机制实现这样的功能？

**Comment by kwindva** [comment]
自造短语功能肯定可以的，就是在custom_phrase.txt添加你要它输出的东西。
![图片](https://github.com/user-attachments/assets/8c66d823-9bc2-4ad9-b120-f9bfa0cf15f5)

**Comment by shidejie** [comment]
您好，我说错了，是自动造词。自动造词能行吗？13:03, 2024年11月25日, "易楚" ***@***.***>:
自造短语功能肯定可以的，就是在custom_phrase.txt添加你要它输出的东西。
default.png (view on web)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
Regards,Shidejie

**Comment by sidottu** [comment]
自己外掛個詞庫吧，很好實現特殊詞的，像我這個就是外掛詞庫實現的。還有些中文中常用的英文詞，就是對於這些不便自動造詞用的
![image](https://github.com/user-attachments/assets/612de88c-9523-4fed-bb1b-552af5d1fd66)

**Reply by yjnu** [reply]
请问下，词库能不能实现换行的词条，不用 lua，我试了词条中加入 \n 没效果

**Reply by fxliang** [reply]
> 请问下，词库能不能实现换行的词条，不用 lua，我试了词条中加入 \n 没效果

目前Rime的文本词库不能直接编码\n这样的内容，这个是用来分词条的，要词典带\n明文，然后用lua来转义成实际的数据才可以

**Reply by yjnu** [reply]
感谢大佬的回复
